docs/application/web/guides/connectivity/bluetooth.md

```diff
@@ -697,8 +757,531 @@ To read and write a value of the Bluetooth descriptor:
    descriptor.writeValue(newValue, writeSuccess, writeFail);
    ```
 
+### Accessing the ATT MTU of connected device
+
+To get the ATT MTU value or request change of the ATT MTU value:
```

Comment:
Suggestion:
To get the ATT MTU value or request change of the ATT MTU value, follow these steps:

Reply:
I have applied your suggestions.